### PR TITLE
URL changes to match new site map

### PR DESCRIPTION
**Pull request by rosskarchner**
Moves /blog, /newsroom, and /about-us to their intended URL, per @schaferjh document

Initial approach was to fix this via server configuration, but this seems like a more sustainable way of doing it...

Also adds some fin-ed-resources URL's that were missing, like the new-ish money-as-you-grow.

## Changes

- change urls.py so that blog,  newsroom, and the-bureau are under /about-us
- adds missing URL's for fin-ed-resources
- change all templates and sheerlike settings to reference the new URL's
- redirects the old URL's to the new ones
- move the-bureau sheer templates under about-us

## Testing

- observe that all links go to new blog and newsroom urls, and that the blog and newsroom (including filter, etc) performs as before.

To test the fin-ed-resources stuff:

- checkout out fin-ed-resources to a sibling directory of cfgov-refresh
- in that directory, `npm install` and `grunt`
- back in cfgov-refresh, `cfgov/manage.py sheer_index`
- `runserver` and observe that you can browse to localhost:8000/money-as-you-grow/

## Review

- @kurtw
- @kave
- @richaagarwal

## Checklist

* [x] Changes are limited to a single goal (no scope creep)
* [x] Code can be automatically merged (no conflicts)
* [x] Code follows the standards laid out in the [front end playbook](https://github.com/cfpb/front-end)
* [x] Passes all existing automated tests
* [ ] New functions include new tests
* [ ] New functions are documented (with a description, list of inputs, and expected output)
* [ ] Placeholder code is flagged
* [ ] Visually tested in supported browsers and devices
* [ ] Project documentation has been updated (including the "Unreleased" section of the CHANGELOG)

### CODE REVIEW

**Comment by rosskarchner** [comment]
Broadening the scope of this PR to include other URL changes/additions

**Comment by KimberlyMunoz** [comment]
I don't feel great about moving `/blog` to `/about-us/blog`. With the current configuration there are few broken links on the blog when we do the switch over.

`http://beta.consumerfinance.gov/blog/mortgage-moves-how-much-can-you-afford/` replaces `http://www.consumerfinance.gov/blog/mortgage-moves-how-much-can-you-afford/` and the slugs should match in most cases. That's not going to be the case when all the beta links point to `/about-us/blog`.

**Comment by rosskarchner** [comment]
@KimberlyMunoz I'm not sure I understand-- what changes about the slugs?

**Comment by KimberlyMunoz** [comment]
Nothing changes about the slugs, they stay the same and so the relative paths used to match up exactly before. With this change, when the switch over happens the relative paths won't match any more.

**Comment by rosskarchner** [comment]
So, you mean, when a blog post links to another blog post, it'll link to /blog/whatever/?

Is it not enough if that gets caught by the redirect, so the user gets transparently bounced to /about-us/blog/whatever/?

**Comment by KimberlyMunoz** [comment]
It's probably fine with the redirect. We lose some SEO link juice depending on the type of redirect, but it's still workable. I don't feel great about it because it seems like an unnecessary change to me. But it's a small concern that I wanted to raise before we do it, but it shouldn't hold it up or stop it. 

Think of this feedback as one of those annoying "Are you sure" pop-up windows.

imgres
:point_up_2:  it me

**Comment by schaferjh** [comment]
It's a good point @KimberlyMunoz. Part of my thinking around this is that we will be moving the blog and newsroom under "About us" in Wagtail eventually (soon, I hope), and at that time all this change to the urls will happen anyway. So it's a redirect now or later kind of thing.

**Comment by rosskarchner** [comment]
So with the attached change, links are fixed at sheer_index time. Is this a terrible idea?

Here are the rules that are applied: https://github.com/cfpb/cfgov-refresh/pull/1689/files#diff-fe16ef71622313a8f4d91e408a59b891R4

**Comment by rosskarchner** [comment]
travis failure noted

File: cfgov/processors/processors_common.py [hunk 1]
```diff
@@ -0,0 +1,32 @@
+from BeautifulSoup import BeautifulSoup
```
Comment (kurtw) [inline cfgov/processors/processors_common.py]: this should be `from bs4 import BeautifulSoup`

**Comment by kurtw** [comment]
After addressing the comment, I think it's good to go.

File: cfgov/processors/processors_common.py (hunk 1 above)
Comment (rosskarchner) [inline cfgov/processors/processors_common.py]: Thanks, kurt-- I swear there were a few days where we had both versions in our requirements.

**Comment by kave** [comment]
hey ross could you rebase this branch to remove alot of the noise from other commits. thanks

**Comment by rosskarchner** [comment]
done!

**Comment by kave** [comment]
:+1:

**Comment by rosskarchner** [comment]
Added one last commit, moving /hmda/ to /data-research/mortgage-data-hmda/

**Comment by kave** [comment]
:+1:

**Pull request merged**